docs/pages/access-controls/device-trust/guide.mdx

```diff
@@ -1,462 +1,212 @@
-Device enrollment is key in correctly identifying a device throughout its
-lifetime. The more trusted the enrollment operator, the better the ongoing
-guarantees that the device itself is trustworthy.
+  Support for other operating systems, access from Web UI and application
+  access is planned for upcoming Teleport versions.
 </Admonition>
 
-The tie between the device inventory and enrollment is a device enrollment
-token. A device enrollment token needs to be created by a device admin and sent
-to the person performing the enrollment ceremony. The token ties the actual
-device being enrolled to the expected device in the inventory, adding an extra
-layer of protection.
-
-## Prerequisites
-
-(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
-
-(!docs/pages/includes/device-trust/prereqs.mdx!)
-
-## Step 1/2. Register a trusted device
+Device Trust requires two of the following steps to have been configured:
 
-The `tctl` tool is used to manage the device inventory. A device admin is
-responsible for managing devices, adding new devices to the inventory and
-removing devices that are no longer in use.
+- Device enforcement mode configured via either a role or a cluster-wide config.
+- Trusted device registered and enrolled with Teleport.
 
-A role that grants permissions for the `device` resource is necessary to manage
-the inventory. Save the yaml below as `device-admin.yaml` and create it in your
-cluster:
+In this guide, you will update an existing user profile to assign the preset `require-trusted-device`
+role and then enroll a trusted device into Teleport to access a resource (a test linux server)
+protected with Teleport.
 
-```yaml
-version: v6
-kind: role
-metadata:
-  name: device-admin
-spec:
-  allow:
-    rules:
-    - resources: ["device"]
-      verbs:
-      - create
-      - read
-      - list
-      - update
-      - delete
-      - create_enroll_token
-      - enroll
-```
-
-```code
-$ tctl create -f device-admin.yaml
-role 'device-admin' has been created
-```
+### Prerequisites
 
-<Admonition type="tip" title="Editor Role">
-For clusters created using Teleport v12 or newer, the builtin role `editor` has
-the necessary permissions to manage devices. For production setups a separate
-role is still recommended.
-</Admonition>
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
 
-Note that in addition to the usual CRUD verbs (create, read, list, update and
-delete), we have also included `create_enroll_token` and `enroll`. The
-`create_enroll_token` verb is necessary to execute the `tctl devices enroll`
-command; `enroll` is necessary to execute `tsh device enroll`. Both are
-demonstrated in the following section.
+(!docs/pages/includes/device-trust/prereqs.mdx!)
 
-Make sure your user has the `device-admin` role:
+- User with `editor` role.
+  - ```code
+    $ tsh status
+    > Profile URL:      https://teleport.example.com:443
+    Logged in as:       myuser
+    Cluster:            teleport.example.com
```

Comment:
Funny I couldn't find it. Can you please share the link to the var definition if you find one.

Reply:
So I have added default sets of variable in https://github.com/gravitational/teleport/pull/30821/commits/deaa15ec835baebb0b3c4e6d0c6f86f594d5da35

But since there are variations in the names we use in other docs, and not to blow the scope of this PR, I have also opened a ticket to discuss that https://github.com/gravitational/teleport/issues/31207